src/metaswitch/crest/tools/sstable_provisioning/README.md

```diff
@@ -2,11 +2,11 @@
 
 These scripts will enable you to create a collection of sstables (Cassandra raw data) and then inject those tables directly into your Cassandra cluster.
 
-All the scripts assume they are being run on a Homer or Homestead node with a correctly configured and balanced Cassandra cluster.
+All the scripts assume they are being run on a Homer or Homestead Cassandra node and that the Cassandra clusters are correctly configured and balanced.
```

Comment:
I wonder if readers will understand what the "Homestead Cassandra node" is. Might be better to say "the node running the Homer or Homestead database".